rts/System/SpringApp.h

```diff
@@ -62,6 +64,9 @@ class SpringApp
 	// this gets passed along to PreGame (or SelectMenu then PreGame),
 	// and from thereon to GameServer if this client is also the host
 	std::shared_ptr<ClientSetup> clientSetup;
+
+	// used to delay relaying messages to console to avoid issues
```

Comment:
Good. Consider
```suggestion
	/* Member instead of global to avoid catching unrelated early messages.
	 * In particular, modes like `spring -p` want a clean output log without library chaff. */
```

Reply:
Done